src/components/ContactButtons.tsx

```diff
@@ -0,0 +1,53 @@
+import { type Contacts } from '@src/models/contacts';
+import { FC } from 'react';
+import Image from 'next/image';
+
+type logoProps = {
+  discord: string;
+  youtube: string;
+  twitch: string;
+  facebook: string;
+  twitter: string;
+  instagram: string;
+  website: string;
+  email: string;
+  other: string;
+};
+
+const logo: logoProps = {
+  discord: '/favicon-16x16.png',
+  youtube: '/nebula-logo.png',
+  twitch: '/nebula-logo.png',
+  facebook: '/nebula-logo.png',
+  twitter: '/nebula-logo.png',
+  instagram: '/nebula-logo.png',
+  website: '/nebula-logo.png',
+  email: '/nebula-logo.png',
+  other: '/Jupiter.png',
+};
+
+const ContactButtons: FC<{ contact: Contacts[] }> = ({ contact }) => {
+  return (
+    <>
+      {contact.map((item, key) => (
+        <button
+          key={key}
```

Comment:
Instead of using the index as a key, use the URL since it should be unique for every item

Reply:
Updated the key to accept url as well as Iclub object to take an array of Contact